modules/plottinggl/src/utils/axisrenderer.cpp

```diff
@@ -122,6 +122,28 @@ AxisRendererBase::AxisRendererBase(const AxisSettings& settings)
 AxisRendererBase::AxisRendererBase(const AxisRendererBase& rhs)
     : settings_{rhs.settings_}, shaders_{getShaders()} {}
 
+AxisRendererBase::AxisRendererBase(AxisRendererBase&& rhs)
```

Comment:
use defualt?

Reply:
with default, the following assertions fail:
```c++
    static_assert(std::is_move_assignable_v<AxisRendererBase>);
    static_assert(std::is_move_assignable_v<AxisRenderer>);
```

Reply:
why? your implementation looks like a default one?

Reply:
or have you skipped some members?

Reply:
using default now after fixing all members.